DittoHealthMetrics/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
-    alias libs.plugins.com.android.library
-    alias libs.plugins.org.jetbrains.kotlin.android
+    id "com.android.library"
+    id "org.jetbrains.kotlin.android"
```

Comment:
What's the reason for moving away from version catalog entries for these? Were they causing an issue?

Reply:
No issues, I just did a copy and paste without updating them to use version catalog.